reo/src/utils.jl

```diff
@@ -235,13 +235,13 @@ Base.@kwdef struct Parameter
      GridExportRates
      FuelBurnSlope
      FuelBurnYInt
-     MaxGridSales::Array{<:Real, 1}
```

Comment:
@zolanaj can we keep this type for MaxGridSales?

Reply:
I thought it could be a good feature to impose different limits between net-metering and wholesale sales.  This would actually be different from the existing formulation, which treats all "1W" destinations the same.

For now, let's keep MaxGridSales under this type, and we can adjust the constraint to fit the original formulation.  I'll push a commit to that end shortly.